### PR TITLE
update menus to show active page

### LINKED ISSUE #534: Should highlight current page in top tabs
No indication of what page you are on by looking at the top bar.

image

### PR DESCRIPTION
**Pull request by glittle**
Closes #534

### CODE REVIEW

**Comment by tonysurma** [comment]
@mheggeseth can you take a peek at this one, @BillWagner same q on merge commits.  thanks

**Comment by BillWagner** [comment]
The merge commits are fine. ( @glittle if you are comfortable rebasing, that shows your diffs easier in a PR. If you want help on how that works, I can show you ).

**Comment by tonysurma** [comment]
okay, then if PR review goes well I will merge

File: AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -1,6 +1,7 @@
-﻿<!DOCTYPE html>
+﻿﻿@using AllReady.Controllers
+<!DOCTYPE html>
 <html lang="en">
-<head>
+﻿<head>
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml]: Somehow a "ZERO WIDTH NO-BREAK SPACE" character (http://stackoverflow.com/questions/9691771/why-is-65279-appearing-in-my-html) got inserted just before the `<head>` tag which is causing the errant white space at the top of the page you see below.
image

File: AllReadyApp/Web-App/AllReady/wwwroot/css/site.css [hunk 2]
```diff
@@ -138,15 +138,23 @@ Navigation
...
-.navbar-default .navbar-nav > li:hover, .navbar-default .navbar-nav > li:active, .navbar-default .navbar-nav > li:focus {
+.navbar-default .navbar-nav > li:hover,
+.navbar-default .navbar-nav > li:active,
+.navbar-default .navbar-nav > li:focus {
     background-color: #ab6627;
 }
 
-.navbar-default .navbar-nav > li > a:hover, .navbar-default .navbar-nav > .active > a, .navbar-default .navbar-nav > .active > a:hover, .navbar-default .navbar-nav > .active > a:focus {
-    background: none;
-    color: #fff;
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/wwwroot/css/site.css]: The deletion of this code makes the root of the Admin menu black on hover even when it's not active. Despite the fact that Chrome thinks white is the prevailing color the `.navbar-default .navbar-nav > li > a:hover` selector from `navbar.less` does in fact have higher specificity (by 1) than `.navbar-default .navbar-nav > li > a` from `site.css`.
image

**Comment by tonysurma** [comment]
@glittle after you respond to the review comments please rebase off of master as we now have a fix for the build/test issues!

For reference on rebasing this page helps (replace upstream with HTBox in examples) https://robots.thoughtbot.com/keeping-a-github-fork-updated

**Comment by glittle** [comment]
Thanks @tonysurma. I don't expect to be able to work on this anytime soon. If someone else can fix up this issue, please do. Another option is to reject this PR and do similar changes again in your own fork.

**Comment by tonysurma** [comment]
Ok, thanks.  We will figure out how to best take that forward.

**Comment by tonysurma** [comment]
@BillWagner @MisterJames any ideas on best practice to pick up an existing PR, make a few changes and then bring it in with minimal rework?  Thanks

**Comment by mheggeseth** [comment]
I'll take care of it. New PR coming forthwith.

**Pull request merged**